draft-ietf-quic-transport.md

```diff
@@ -1741,6 +1741,15 @@ parameters as a connection error of type TRANSPORT_PARAMETER_ERROR.
 Endpoints use transport parameters to authenticate the negotiation of
 connection IDs during the handshake; see {{cid-auth}}.
 
+Application Layer Protocol Negotiation (ALPN; see {{?ALPN=RFC7301}}) allows
+clients to offer multiple application protocols during connection
+establishment. The transport parameters that a client includes during the
+handshake apply to all application protocols that the client offers. Application
+protocols can recommend values for transport parameters, such as the initial
+flow control limits. However, application protocols that require values for
```

Comment:
```suggestion
flow control limits. However, application protocols that require specific values for
```

I think we are talking about the requirement of values rather than if TP take values.

Reply:
Or
```suggestion
flow control limits. However, application protocols that constrain values for
```

Reply:
Or

```suggestion
flow control limits. However, application protocols that set constraints on values for
```